examples/ConsoleAppWithFailOver/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.ConsoleApplicationWithFailOver
+{
+    class Program
+    {
+        static IConfiguration? Configuration { get; set; }
+        private static IConfigurationRefresher? _refresher;
+
+        static void Main(string[] args)
+        {
+            Configure();
+
+            var cts = new CancellationTokenSource();
+
+            _ = Run(cts.Token);
+
+            // Finish on key press
+            Console.ReadKey();
+            cts.Cancel();
+        }
+
+        private static void Configure()
+        {
+            var builder = new ConfigurationBuilder();
+
+            // Load a subset of the application's configuration from a json file and environment variables
+            builder.AddJsonFile("appsettings.json")
+                   .AddEnvironmentVariables();
+
+            IConfiguration configuration = builder.Build();
+
+            IConfigurationSection endpointsSection = configuration.GetSection("AppConfig:Endpoints");
+            IEnumerable<Uri> endpoints = endpointsSection.GetChildren().Select(endpoint => new Uri(endpoint.Value));
+
+            if (endpoints == null || !endpoints.Any())
+            {
+                Console.WriteLine("Endpoints not found.");
+                Console.WriteLine("Please set the array 'Appconfig:Endpoints' in appsettings.json with valid Azure App Configuration replica endpoints and re-run this example.");
+                return;
+            }
+
+            // Augment the configuration builder with Azure App Configuration
+            // Pull the connection string from an environment variable
+            builder.AddAzureAppConfiguration(options =>
+            {
+                options.Connect(endpoints, new DefaultAzureCredential())
+                       .ConfigureRefresh(refresh =>
```

Comment:
I'd recommend keeping the example as simple as possible since it's just to demonstrate failover. Can just load config and display a value.

Reply:
Done!